backend/coins/coin/coin.go

```diff
@@ -41,6 +41,12 @@ type Coin interface {
 	// The fee unit is usually the same as the main unit, but can differ.
 	Unit(isFee bool) string
 
+	// SetformatUnit sets the unit used to format the amount, e.g. "BTC" or "SATS".
```

Comment:
explain what it does for non-BTC accounts. does it even need to be in the interface if it is only used for BTC? consider removing it from the interface.

Reply:
Removed, thx